http_disk_server.py

```diff
@@ -88,6 +90,10 @@ def check_bindable(port):
 def eprint(str):
     print(OUTPUT_PREFIX + str, file=sys.stderr)
 
+def dprint(str):
+    if VERBOSE:
+        print(OUTPUT_PREFIX + str, file=sys.stderr)
```

Comment:
You can call `eprint` directly I think

Reply:
fixed in [966919e](https://github.com/xcp-ng/http-nbd-transfer/pull/14/commits/966919e8f056c44b482f5e6e02a6e7d3e9d5243c)